src/main/java/com/xjeffrose/xio/client/loadbalancer/Distributor.java

```diff
@@ -74,34 +70,46 @@ public void stop() {
 
   /**
    * The vector of pool over which we are currently balancing.
+   *
+   * @return Node list
```

Comment:
We should probably find a tool to generate javadocs so we can make future diffs more readable.

Reply:
ya.  i normally make format/style changes in separate prs, but this one got away with me.  what's weird is it only gives errors if you've started creating a javadoc but it is invalid.  doesn't say anything about stuff that has no javadoc comments.  i've found that javadoc comments are only useful if they provide context and maybe examples of usage...